mmv1/products/clouddeploy/DeployPolicy.yaml

```diff
@@ -175,71 +175,19 @@ properties:
                 - name: oneTimeWindows
                   type: Array
                   description: "Optional. One-time windows within which actions are restricted."
+                  custom_expand: "templates/terraform/custom_expand/clouddeploy_policy_onetime_windows.go.tmpl"
+                  custom_flatten: "templates/terraform/custom_flatten/clouddeploy_policy_onetime_windows.go.tmpl"
                   item_type:
                     type: NestedObject
                     properties:
-                      - name: "startDate"
-                        type: NestedObject
-                        description: "Required. Start date."
+                      - name: "start"
+                        type: String
+                        description: "Required. Start date and time in the datetime format, e.g. 2024-01-02 09:00:00."
                         required: true
-                        properties:
-                          - name: "year"
-                            type: Integer
-                            description: "Year of the date. Must be from 1 to 9999, or 0 to specify a date without a year."
-                          - name: month
-                            type: Integer
-                            description: "Month of a year. Must be from 1 to 12, or 0 to specify a year without a month and day."
-                          - name: day
-                            type: Integer
-                            description: "Day of a month. Must be from 1 to 31 and valid for the year and month, or 0 to specify a year by itself or a year and month where the day isn't significant."
-                      - name: "endDate"
-                        type: NestedObject
-                        description: "Required. End date."
+                      - name: "end"
+                        type: String
+                        description: "Required. End date and time,  in the datetime format, e.g. 2025-01-02 09:00:00."
```

Comment:
no need to include required in the description the generator includes it in the docs automatically when a field has `required: true`